### PR TITLE
Translate strings being passed to the `page_header` function

### PR DESCRIPTION
**Pull request by smcmurtry**
Wrapped strings being passed to letter branding everywhere except letter pages. Still need to add translations to .po files. Will do that (and cconvert from draft) after some more of the translation PRs are in to avoid merge conflicts.

### CODE REVIEW

**Comment by smcmurtry** [comment]
@anikbrazeau would you mind translating the following:
"This will change the mobile number for ___" (user name)
"Change organisation name"
"Link service to organisation"
"No organisations"
"Set email branding"
"Change email branding"
"Choose the branding you’d like on your emails."
"You need to choose an option"
'We’ll email you once your branding’s ready to use, or if we need any more information."
"Confirm changes"
"Change your ____" (email address, name, etc)
"Change your password"
"We’ve sent a security code to your new ___" (email address, phone number, etc)

**Comment by smcmurtry** [comment]
Actually I can fill in one of these from https://github.com/cds-snc/notification-api/issues/779

"Change your ___" => "Changez votre ___"

**Comment by anikbrazeau** [comment]
"This will change the mobile number for ___" (user name)
**FR = "Ceci changer le numéro de téléphone cellulair de :**

"Change organisation name"
**FR = "Changer le nom de l'organisation**

"Link service to organisation"
**FR = "Lier le service à l'organisation"**

"No organisations"
**FR = "Aucune organisation"**

"Set email branding"
**FR = Définir l'image de marque du courriel**

"Change email branding"
**FR = Changer l'image de marque du courriel**

"Choose the branding you’d like on your emails."
**FR = "Choisir la marque que vous souhaitez pour vos courriels.**

"You need to choose an option"
**FR = Vous devez sélectionner une option**

'We’ll email you once your branding’s ready to use, or if we need any more information."
**FR = Nous vous enverrons un courriel dès que votre marque sera prête à être utilisé, ou bien si nous avons besoin plus d'informations.**

"Confirm changes"
**FR = Confirmer les modifications**

"Change your ____" (email address, name, etc)
**FR = "Changer votre ____" (adresse courriel, nom, etc.)**

"Change your password"
**FR = "Changer votre mot de passe"**

"We’ve sent a security code to your new ___" (email address, phone number, etc)
**FR = "Nous avons envoyé un code de sécurité à votre nouvelle** (adresse courriel)
**FR = "Nous avons envoyé un code de sécurité à votre nouveau** (numéro de téléphone)

For this last one ... if "new" is not essential to the context, perhaps we can remove it?
**EN = "We’ve sent a security code to your ___" (email address, phone number, etc)**
**FR = "Nous avons envoyé un code de sécurité à votre ** (adresse courriel, numéro de téléphone, etc.)

File: app/templates/views/manage-users/edit-user-mobile.html [hunk 1]
```diff
@@ -11,11 +11,11 @@
 {% block maincolumn_content %}
 
   {{ page_header(
-    'Change team member’s mobile number',
+    _('Change team member’s mobile number'),
     back_link=url_for('main.edit_user_permissions', service_id=current_service.id, user_id=user.id)
   ) }}
 
-  <p id="user_name">This will change the mobile number for {{ user.name }}.</p>
+  <p id="user_name">{{ _('This will change the mobile number for ') + user.name }}.</p>
```
Comment (sastels) [inline app/templates/views/manage-users/edit-user-mobile.html:18]: It's easy to pass variables into babel (not needed here, admittedly, but good if the variable is in the middle of a sentence)

File: app/translations/fr/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -4704,9 +4710,14 @@ msgid "This will change the email address for"
 msgstr "Cela changera l’adresse courriel pour "
 
 #: app/templates/views/manage-users/edit-user-mobile.html:8
+#: app/templates/views/manage-users/edit-user-mobile.html:14
 msgid "Change team member’s mobile number"
 msgstr "Changer le numéro de téléphone cellulaire d’un membre de l’équipe"
 
+#: app/templates/views/manage-users/edit-user-mobile.html:18
+msgid "This will change the mobile number for "
+msgstr "Ceci changer le numéro de téléphone cellulair de : "
```
Comment (sastels) [inline app/translations/fr/LC_MESSAGES/messages.po:4719]: ":" for the French but not the English?

File: app/translations/fr/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -5108,6 +5142,27 @@ msgstr ""
...
+msgid "You need to choose an option"
+msgstr "Vous devez sélectionner une option"
+
+#: app/templates/views/service-settings/branding/email-options.html:41
+msgid ""
+"We’ll email you once your branding’s ready to use, or if we need any\n"
+"      more information."
+msgstr ""
+"Nous vous enverrons un courriel dès que votre marque sera prête à être\n"
+"      utilisé, ou bien si nous avons besoin plus d'informations."
```
Comment (sastels) [inline app/translations/fr/LC_MESSAGES/messages.po:5164]: Why the strange indenting?

File: app/translations/fr/LC_MESSAGES/messages.po (hunk 2 above)
Comment (smcmurtry) [reply app/translations/fr/LC_MESSAGES/messages.po:4768]: Yes that is what @anikbrazeau wrote above.

File: app/templates/views/manage-users/edit-user-mobile.html (hunk 1 above)
Comment (smcmurtry) [reply app/templates/views/manage-users/edit-user-mobile.html:18]: Thanks! I'll definitely be using that in the future.

File: app/translations/fr/LC_MESSAGES/messages.po (hunk 3 above)
Comment (smcmurtry) [reply app/translations/fr/LC_MESSAGES/messages.po:5213]: Not sure, that's how pybabel indented the english text so I just copied the format for FR

**Pull request merged**